pom.xml

```diff
@@ -94,6 +94,13 @@
     </repository>
   </repositories>
 
+  <pluginRepositories>
```

Comment:
was this added on purpose? don't think we should be depending on any snapshot dependencies.

Reply:
just read the commit message, I think 1.0.1 is published.

Reply:
Hmm, doesn't seem so:
` Could not find artifact co.cask:cdap-maven-plugin:jar:1.0.1`

Reply:
hm 1.0.0 is there. I would check with @sreevatsanraman with what the difference in 1.0.1-SNAPSHOT is, we really shouldn't have any snapshot version anywhere.